### PR TITLE
Fixed micsLocked check

### PR DESCRIPTION
**Pull request by BobakOftadeh**
The micsLocked boolean is now set correctly when microphones are locked.

### CODE REVIEW

**Comment by antobinary** [comment]
In the logs I also spot ```W20181127-16:09:46.457(0)? (STDERR) error: CLIENT LOG: TypeError: Cannot read property 'lockSettingsProp' of undefined
W20181127-16:09:46.457(0)? (STDERR)     at VideoService.isLocked (http://anton2```

Could you please try to handle it in this PR too?

**Comment by antobinary** [comment]
How do we test this fix?

**Comment by BobakOftadeh** [comment]
Lock the webcams, then join the meeting as moderator and try to join audio using the join audio button.

File: bigbluebutton-html5/imports/ui/components/video-provider/service.js [hunk 1]
```diff
@@ -106,7 +106,8 @@ class VideoService {
 
   isLocked() {
     const m = Meetings.findOne({ meetingId: Auth.meetingID });
-    return m.lockSettingsProp ? m.lockSettingsProp.disableCam : false;
+    const lockSetings = m.lockSettingsProp || {};
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/video-provider/service.js]: The error will still happen, you need to check if there is a meeting.

**Pull request merged**